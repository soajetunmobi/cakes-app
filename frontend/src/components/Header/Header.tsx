import React, { FunctionComponent } from 'react'

export const Header: FunctionComponent = () => {
  return (
    <div className='header'>
      <h1 className='logo'>Cake Reviews</h1>
    </div>
  )
}
