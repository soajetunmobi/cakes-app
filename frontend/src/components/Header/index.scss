.header {
    display: flex;
    align-items: flex-start;
    padding-left: 20px;
}

.logo {
    font-size: 2rem;
    text-align: center;
    line-height: var(--logo-height);
    height: var(--logo-height);
    margin: 0;
}
