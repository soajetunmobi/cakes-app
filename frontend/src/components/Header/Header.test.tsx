import React from 'react'
import renderer from 'react-test-renderer'
import { Header } from './Header'

test('it renders correctly', () => {
  const root = renderer.create(<Header />).toJSON()
  expect(root).toMatchSnapshot()
})
