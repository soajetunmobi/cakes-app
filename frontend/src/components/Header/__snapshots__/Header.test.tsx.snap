// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
<div
  className="header"
>
  <h1
    className="logo"
  >
    Cake Reviews
  </h1>
</div>
`;
